### PR TITLE
Add refresh-releases subcommand, etc.

### LINKED ISSUE #96: update releases/<profile>.yaml based on what's really there
Perhaps the new `release` subcommand could also traverse the regions and update `releases/<profile>.yaml` with the matching AMIs that actually exist there.

Or, the same broker mechanism used in `release` could be used in a different subcommand.

No/few changes to the code to update `releases/README.md` should be necessary as long as the release's YAML is fully fleshed out.

### PR DESCRIPTION
**Pull request by tomalok**
* **builder.py**
  + gen-release-readme
    - convert `build_time` to int
  + release
    - add `source_region` to copied AMI tags
    - check source AMI's permissions, queue for fixing, if necessary
  + refresh-releases
    - update releases/<profile>.yaml based on AMIs that exist in regions

* Release Alpine 3.12.2 & today's edge

Resolves #96, related to #95

### CODE REVIEW

File: scripts/builder.py [hunk 1]
```diff
@@ -42,6 +42,7 @@
 import textwrap
 import subprocess
 import urllib.error
+import dateutil.parser
```
Comment (mcrute) [inline scripts/builder.py:45]: This will have to be added to the python dependency list.

**Review by mcrute — CHANGES_REQUESTED** [review]
Looks pretty good to me, just one comment.

I really like the approach of using the image tags as a database for all the things we care about and just generating the release manifests from that. I think it would be pretty nice if we could just publish all of those artifacts to an S3 bucket with web hosting enabled so we could stop storing them in this git repo. But that's a topic for a different feature request.

File: scripts/builder.py (hunk 1 above)
Comment (tomalok) [reply scripts/builder.py:45]: hm, wonder how `dateutil.parser` actually worked for me 'til now?

Assuming this is the dependency list, and without adding it to ...
```
    subprocess.run(["build/bin/pip", "install", "-U", "pip",
        "pyhocon", "boto3", "PyYAML"])
```
...it somehow gets installed into *build/lib/python3.9/site-packages/dateutil/* just fine?

Comment (mcrute) [reply scripts/builder.py:45]: Hrm... I would guess it gets pulled in by either botocore or pyhocon. It's still better to be explicit about our direct dependencies should we for some reason drop the one that pulls it in transitively later.

Comment (tomalok) [reply scripts/builder.py:45]: added explicit `python-dateutils` dependency.

**Pull request merged**